### PR TITLE
[MIG] website_logo: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
Website logo
============

Load a logo image to be used on website only. This allows to use an internal
company logo (for reports) and a different website logo.

cc @Tecnativa

### CODE REVIEW

File: website_logo/README.rst [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
+company logo (for reports) and a different website logo.
 
 Configuration
 =============
 
 To configure this module, you need to:
 
-* go to your company and load a logo image in the website logo field
+* Go to 'Website Admin'.
+* In the configuration form of a website there is a category logo.
```
Comment (pedrobaeza) [inline website_logo/README.rst]: s/category logo/"Logo" group

File: website_logo/README.rst [hunk 2]
```diff
@@ -1,21 +1,37 @@
...
+* Load a logo image in the logo field.
+
+Usage
+=====
+
+This addon supports multi-website. If no logo is defined a transparent image is
```
Comment (pedrobaeza) [inline website_logo/README.rst]: comma after `defined`.

File: website_logo/__manifest__.py [hunk 3]
```diff
@@ -1,40 +1,30 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     'name': 'Website logo',
     'summary': 'Website company logo',
-    'version': '8.0.1.0.0',
+    'version': '10.0.1.0.0',
     'category': 'Website',
-    'author': "Agile Business Group,Odoo Community Association (OCA)",
+    'author': "Agile Business Group, "
+              "Antiun Ingeniería S.L., "
```
Comment (pedrobaeza) [inline website_logo/__manifest__.py]: Remove Antiun

**Review by pedrobaeza — APPROVED** [review]
Little changes, but working on runbot

File: website_logo/README.rst (hunk 2 above)
Comment (chienandalu) [reply website_logo/README.rst]: Not really mandatory in English and it would kind of hinder the reading of the whole phrase...

**Comment by chienandalu** [comment]
@pedrobaeza changes done and squashed

**Comment by eLBati** [comment]
Thanks @chienandalu 
Commit ae19202b193c8c828474b8ce79e07ee4d029cdf3 author is not recognized.
Is ```Stephan Rozendaal <stephan@west.nl>``` correct?

Then, :+1:

**Comment by pedrobaeza** [comment]
@eLBati that is because the GitHub user doesn't have that email address linked to its account.

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Comment by eLBati** [comment]
@pedrobaeza so he needs to configure the github account, to fix https://github.com/OCA/website/pull/357/commits/ae19202b193c8c828474b8ce79e07ee4d029cdf3 , right?

**Comment by pedrobaeza** [comment]
Exactly

**Comment by StephanRozendaal** [comment]
Hi, I added stephan@west.nl to my account.

**Comment by eLBati** [comment]
@StephanRozendaal thanks

**Review by eLBati — APPROVED** [review]

File: website_logo/README.rst (hunk 2 above)
Comment (pedrobaeza) [reply website_logo/README.rst]: Yes, it's mandatory in English (http://www.shertonenglish.com/resources/es/conditionals/type0.php)

Comment (lasley) [reply website_logo/README.rst]: Native English speaker - can confirm, comma required

Comment (chienandalu) [reply website_logo/README.rst]: Ok, I had the rule wrong on my mind (comma is not needed when the clause comes before the condition). Anyway, doesn't it feel like the sentence has more commas than needed? :smiley:

**Review by lasley — APPROVED** [review]

**Comment by lasley** [comment]
Looks good @chienandalu, thanks! Can you please squash f9a9c56 & 7dfd73c

FYI Travis was failing on one, but I'm pretty sure it was a timeout. Re-running it

**Comment by chienandalu** [comment]
Thanks, @lasley  :smile:

**Pull request merged**